src/Algebra/Properties/CommutativeMonoid.agda

```diff
@@ -23,3 +25,18 @@ open CommutativeMonoid M
   ; assoc     to +-assoc
   ; comm      to +-comm
   )
+
+private variable
+  x : Carrier
+
+leftInv→rightInv : LeftInvertible _≈_ 0# _+_ x → RightInvertible _≈_ 0# _+_ x
```

Comment:
These names should be of the form `invertibleˡ⇒invertibleʳ` to be consistent with other library naming schemes.